src/schema/objects/columns.yaml

```diff
@@ -308,16 +308,21 @@ onset:
     acquired data point.
   type: number
   unit: s
-orientation_component:
-  name: orientation_component
-  display_name: Orientation Component
+component:
+  name: component
+  display_name: Component
   description: |
-    Description of the orientation of the channel.
+    Description of the spatial axis or component associated with the channel.
   type: string
   enum:
     - x
     - y
     - z
+    - quat_x
+    - quat_y
+    - quat_z
+    - quat_w
```

Comment:
there is currently no description in the spec of what `quat_*` would refer to. I think I would try to add this as condensed as possible into the description above.

> Description of the spatial axis or component associated with the channel.

Something like:

> Description of the spatial axis or component associated with the channel. For example `x`, `y`, or `z` for the axes in a Cartesian coordinate system, or `quat_x` ...